### PR TITLE
Adding 4 rules from MITRE's Center for Threat Informed Defense

### PR DESCRIPTION
**Pull request by RobertSchull**
### Summary of the Pull Request
Summiting the Pyramid is a project from MITRE's Center for Threat Informed Defense and during the course of our research we developed a few analytics that we wanted to make them more accessible. 
link to project page: https://center-for-threat-informed-defense.github.io/summiting-the-pyramid/overview/

### Changelog
new: Service Registry Key Read Access Request
new: Scheduled Task Created - FileCreation
new: Scheduled Task Created - Registry

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by nasbench** [comment]
Thanks for the submission @RobertSchull. I've made some changes to the rules so that they conform to the Sigma HQ standard. 2 main changes are below

- Deleted the Adfind rule as its covered already here https://github.com/SigmaHQ/sigma/blob/2a463fb7ea43e39a76960a745b72aa27618af9fa/rules/windows/process_creation/proc_creation_win_pua_adfind_susp_usage.yml
- I reduced the level of the other rules to low as they are more common and will produce a lot of events.

**Review by nasbench — APPROVED** [review]

File: rules/windows/file/file_event/file_event_win_scheduled_task_creation.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title: Scheduled Task Created - FileCreation
+id: a762e74f-4dce-477c-b023-4ed81df600f9
+status: experimental
+description: Detects the creation of scheduled task.
```
Comment (phantinuss) [inline rules/windows/file/file_event/file_event_win_scheduled_task_creation.yml:4]:
```suggestion
description: Detects the creation of a scheduled task via file creation.
```

File: rules/windows/file/file_event/file_event_win_scheduled_task_creation.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    category: file_event
+detection:
+    selection:
+        TargetFilename|contains:
+            - ':\Windows\System32\Tasks\'
+            - ':\Windows\SysWOW64\Tasks\'
+            - ':\Windows\Tasks\'
+    condition: selection
+falsepositives:
+    - Likely
```
Comment (phantinuss) [inline rules/windows/file/file_event/file_event_win_scheduled_task_creation.yml:28]:
```suggestion
    - Normal behaviour on Windows
```

File: rules/windows/registry/registry_event/registry_event_scheduled_task_creation.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    product: windows
+    category: registry_event
+detection:
+    selection:
+        TargetObject|contains:
+            - '\Microsoft\Windows NT\CurrentVersion\Schedule\TaskCache\Tasks\'
+            - '\Microsoft\Windows NT\CurrentVersion\Schedule\TaskCache\Tree\'
+    condition: selection
+falsepositives:
+    - Likely
```
Comment (phantinuss) [inline rules/windows/registry/registry_event/registry_event_scheduled_task_creation.yml:27]:
```suggestion
    - Normal behaviour on Windows
```

File: rules/windows/registry/registry_event/registry_event_scheduled_task_creation.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+title: Scheduled Task Created - Registry
+id: 93ff0ceb-e0ef-4586-8cd8-a6c277d738e3
+status: experimental
+description: Detects the creation of a scheduled task.
```
Comment (phantinuss) [inline rules/windows/registry/registry_event/registry_event_scheduled_task_creation.yml:4]:
```suggestion
description: Detects the creation of a scheduled task via Registry keys.
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**